README.md

```diff
@@ -144,6 +144,22 @@ To build both the client and the server, just run
 npm run build
 ```
 
+#### Imprint and privacy notices
+
+Links to imprint and privacy notices can be included into the client, if the environment variable
+
+```bash
+export REACT_APP_EOP_IMPRINT="https://example.tld/imprint/"
+export REACT_APP_EOP_PRIVACY="https://example.tld/privacy/"
+```
+
+are set when building the app.
+When building the client via docker these env vars can be set by defining `build-args`
+
+```bash
+docker build --build-arg "EOP_IMPRINT=https://example.tld/imprint/" --build-arg "EOP_PRIVACY=https://example.tld/privacy/" -f docker/client.dockerfile . -t "some-tag"
```

Comment:
Is there a specific reason for setting these via (differently named) build args, instead of just using the exact same environment variables? That would be less confusing to me.

Reply:
No good reason for it. Will change it.